### PR TITLE
Cancel appointment parser

### PR DESCRIPTION
**Pull request by SimWPEric**
To be merged after #83 and #86

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/CancelCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    /**
+     * Parses the given {@code String} of arguments in the context of the CancelCommand
+     * and returns a CancelCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public CancelCommand parse(String args) throws ParseException {
+        try {
+            Index index = ParserUtil.parseIndex(args);
+            return new CancelCommand(index);
+        } catch (ParseException pe) {
```
Comment (quzhetao01) [inline src/main/java/seedu/address/logic/parser/CancelCommandParser.java:23]: good exception handling

**Review by quzhetao01 — APPROVED** [review]
LGTM!

File: src/test/java/seedu/address/logic/parser/CancelCommandParserTest.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    private CancelCommandParser parser = new CancelCommandParser();
+
+    @Test
+    public void parse_validArgs_returnsCancelCommand() {
+        assertParseSuccess(parser, "1", new CancelCommand(INDEX_FIRST_PERSON));
+    }
+
+    @Test
+    public void parse_invalidArgs_throwsParseException() {
```
Comment (Sheeepen) [inline src/test/java/seedu/address/logic/parser/CancelCommandParserTest.java:29]: good naming of the test

**Review by Sheeepen — APPROVED** [review]
LGTM. Clear in naming of command.

**Pull request merged**